include/QCMethod/FukuiDysonAnalysis.hpp

```diff
@@ -0,0 +1,72 @@
+// This file is part of GQCG-gqcp.
+//
+// Copyright (C) 2017-2019  the GQCG developers
+//
+// GQCG-gqcp is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// GQCG-gqcp is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with GQCG-gqcp.  If not, see <http://www.gnu.org/licenses/>.
+//
+#pragma once
+
+#include "Basis/SingleParticleBasis.hpp"
+#include "CISolver/CISolver.hpp"
+#include "FockSpace/ProductFockSpace.hpp"
+#include "HamiltonianBuilder/FCI.hpp"
+#include "Operator/SecondQuantized/SQHamiltonian.hpp"
+#include "RDM/RDMCalculator.hpp"
+
+
+namespace GQCP {
+namespace QCMethod {
+
+
+/**
+ *  A class that solves the FCI Hamiltonian for a given molecule and performs Fukui and Dyson analysis
```

Comment:
```suggestion
 *  A class that diagonalizes the FCI Hamiltonian for a given molecule and performs Fukui and Dyson analysis
```

explain in detail what is meant by 'Fukui and Dyson analysis'